server/src/test/java/org/apache/uniffle/server/storage/MultiStorageManagerTest.java

```diff
@@ -56,6 +57,44 @@ public void selectStorageManagerTest() {
     assertTrue((manager.selectStorage(event) instanceof HdfsStorage));
   }
 
+  @Test
+  public void testStorageManagerSelectorOfPreferCold() {
+    ShuffleServerConf conf = new ShuffleServerConf();
+    conf.setLong(ShuffleServerConf.FLUSH_COLD_STORAGE_THRESHOLD_SIZE, 10000L);
+    conf.set(ShuffleServerConf.RSS_STORAGE_BASE_PATH, Arrays.asList("test"));
+    conf.setLong(ShuffleServerConf.DISK_CAPACITY, 10000L);
+    conf.setString(ShuffleServerConf.RSS_STORAGE_TYPE, StorageType.MEMORY_LOCALFILE_HDFS.name());
+    conf.setString(ShuffleServerConf.MULTISTORAGE_FALLBACK_STRATEGY_CLASS,
+        RotateStorageManagerFallbackStrategy.class.getCanonicalName());
+    conf.set(
+        ShuffleServerConf.MULTISTORAGE_SELECTOR_COLD_STORAGE_PREFER,
+        StorageManagerSelector.ColdStoragePreferredFactor.HUGE_PARTITION
```

Comment:
I am not sure is there any problem if change this to conf.setString('rss.server.multistorage.cold.storage.preferred.factor', "HUGE_PARTITION").

Reply:
Could you describe more about your concern?

Reply:
I worry too much, please ignore it.